install_bootstrap

```diff
@@ -20,15 +20,36 @@ MSG
 function template()
 {
     mkdir -p config
+
+    NGINX_CONF_DIRECTORY="config"
+    NGINX_HTML2_DIRECTORY="config"
+    WEBSERVICE_DIRECTORY="config"
+
+    if [ $IS_FARGATE_DEPLOY == "true" ]; then
+        mkdir -p config/webservice
+        mkdir -p config/nginx-conf
+        mkdir -p config/nginx-html2
```

Comment:
Just curious, why not `nginx-html`? Is the `2` a convention?

Reply:
I called it `html2` because that's the directory name for those files in the nginx container: https://github.com/dockstore/compose_setup/blob/dee2563bcc0db1b876d12751d6fc1d9e195163b4/templates/Dockerfile_ui2.template#L14